VAMobile/src/components/FormWrapper/FormFields/RadioGroup.tsx

```diff
@@ -128,9 +128,8 @@ const RadioGroup = <T,>({ options, value, onChange, disabled = false, error, isR
       const textLines: Array<TextLine> = [{ text: option.labelKey, variant: 'VASelector', color: disabled ? 'checkboxDisabled' : 'primary' }]
 
       if (option.additionalLabelText && option.additionalLabelText.length > 0) {
-        textLines[0].variant = 'MobileBodyBold'
         option.additionalLabelText.forEach((item) => {
-          textLines.push({ text: item, variant: 'MobileBody' })
+          textLines.push({ text: item, variant: 'HelperText' })
```

Comment:
Doesn't this affect the gender identity screen and mailing address validation screen? Is this change acceptable for those two screens?

Reply:
I searched the app for `additionalLabelText` and it's not used anywhere except in PrescriptionHistory, so this change shouldn't affect anything else in the app. The areas you mention use textLines, but they don't use additionalLabelText